node/src/components/storage/disjoint_sequences.rs

```diff
@@ -299,6 +299,12 @@ mod tests {
 
     use super::*;
 
+    fn new_sequence(a: u64, b: u64) -> Sequence {
+        let (low, high) = if a <= b { (a, b) } else { (b, a) };
+        assert!(low <= high);
+        Sequence { low, high }
+    }
+
```

Comment:
Minor nit: personally I'd prefer this to be a constructor defined in the `test` module, like so:
```rust
    use super::*;

    impl DisjointSequences {
        fn new(a: u64, b: u64) -> Sequence {
            let (low, high) = if a <= b { (a, b) } else { (b, a) };
            assert!(low <= high);
            Sequence { low, high }
        }
    }
```
That could even let us keep the `new_should_order_elements()` test.

But surely not a blocker.